bundle/edu.gemini.itc.web/src/main/resources/servlets/web/ITCgnirs.html

```diff
@@ -403,38 +403,34 @@
 			</tr>
 			<tr>
 				<td colspan="2">Read mode set for:  </td>
-
 			</tr>
 			<tr>
-				<td align="right"><input value="VERY_FAINT" name="ReadMode" type="radio" />  </td>
-				<td>very low background, typically for 1-2.5 µm observations of very faint targets</td>
+				<td align="right"> <input value="VERY_FAINT" name="ReadMode" type="radio" />
+				very low background, typically for 1-2.5 µm observations of very faint targets</td>
 			</tr>
 			<tr>
-				<td align="right"><input value="FAINT" name="ReadMode" checked="checked" type="radio" />  </td>
-
-				<td>low background, typically for 1-2.5 µm observations of faint targets</td>
+				<td align="right"><input value="FAINT" name="ReadMode" checked="checked" type="radio" />
+					low background, typically for 1-2.5 µm observations of faint targets</td>
 			</tr>
 			<tr>
-				<td align="right"><input value="BRIGHT" name="ReadMode" type="radio" />  </td>
-				<td>medium background / bright objects, typically for 1-2.5 µm observations of bright targets/standards</td>
+				<td align="right"><input value="BRIGHT" name="ReadMode" type="radio" />
+				medium background / bright objects, typically for 1-2.5 µm observations of bright targets/standards</td>
 			</tr>
 			<tr>
-
-				<td align="right"><input value="VERY_BRIGHT" name="ReadMode" type="radio" />  </td>
-				<td>high background (i.e. thermal IR), typically for 3-5 µm observations</td>
+				<td align="right"><input value="VERY_BRIGHT" name="ReadMode" type="radio" />
+				high background (i.e. thermal IR), typically for 3-5 µm observations</td>
```

Comment:
Does this look right in the browser? If you're shrinking the table down to one column per row (whereas there were two before) and right-aligning, then it seems to me that the radio buttons wouldn't be aligned. I would think you'd have to `<td align="left">` to make this look right?